src/helper/streaming/sse.ts

```diff
@@ -55,8 +55,6 @@ const run = async (
     } else {
       console.error(e)
     }
-  } finally {
-    stream.close()
```

Comment:
Removing these lines means you ignore the following PR.

https://github.com/honojs/hono/pull/2529